include/oneapi/tbb/concurrent_hash_map.h

```diff
@@ -158,7 +158,8 @@ class hash_map_base {
         if (is_initial) {
             init_buckets_impl(ptr, sz);
         } else {
-            init_buckets_impl(ptr, sz, reinterpret_cast<node_base*>(rehash_req_flag));
+            node_base* bucket_initializer = reinterpret_cast<node_base*>(rehash_req_flag);
+            init_buckets_impl(ptr, sz, bucket_initializer);
```

Comment:
```suggestion
            init_buckets_impl(ptr, sz, /* bucket_initializer = */ reinterpret_cast<node_base*>(rehash_req_flag));
```

Reply:
`reinterpret_cast` will produce rvalue of type `node_base*`. Suggested change will not compile